src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+
 public class WorkWithFile {
+    private static final String SPLIT_CHARACTER = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        ArrayList<String> dataFromFile = readCsvFile(fromFileName);
+
+        int operationSupply = 0;
+        int operationBuy = 0;
+
+        for (String line : dataFromFile) {
+            String[] splitLine = line.split(SPLIT_CHARACTER);
+            if (splitLine[0].equals("supply")) {
+                operationSupply += Integer.parseInt(splitLine[1]);
+            }
+            if (splitLine[0].equals("buy")) {
+                operationBuy += Integer.parseInt(splitLine[1]);
+            }
+        }
+        int result = operationSupply - operationBuy;
+
+        String writingData = createString(operationSupply, operationBuy, result);
+        writeToFile(toFileName, writingData);
+    }
+
+    private String createString(int operationSupply, int operationBuy, int result) {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("supply,").append(operationSupply).append(System.lineSeparator());
+        stringBuilder.append("buy,").append(operationBuy).append(System.lineSeparator());
+        stringBuilder.append("result,").append(result);
+        return stringBuilder.toString();
+    }
+
+    private ArrayList readCsvFile(String pathToFile) {
```

Comment:
```suggestion
    private List readCsvFile(String pathToFile) {
```
завжди краще використовувати абстраткний тип щоб можна було підставити туди різні імплементації